### PR TITLE
Update functions table

### PR DESCRIPTION
**Pull request by DoktorSAS**
Fixed some wrong named functions added some missing one

### CODE REVIEW

File: src/client/game/scripting/function_tables.cpp [hunk 1]
```diff
@@ -1872,7 +1872,7 @@ namespace scripting
 		{"_meth_843f", 0x843F}, // SP 0x1405D92F0 MP 0x14032D4E0
 		{"_meth_8440", 0x8440}, // SP 0x1404639C0 MP 0x140562210
 		{"_meth_8441", 0x8441}, // SP 0x140462EA0 MP 0x1405616E0
-		{"_meth_8442", 0x8442}, // SP 0x1402A36A0 MP 0x140358B60
+		{"linkto", 0x8442}, // SP 0x1402A36A0 MP 0x140358B60
```
Comment (mjkzy) [inline src/client/game/scripting/function_tables.cpp:1875]: Don't we already have this? Correct me if wrong.

Comment (DoktorSAS) [reply src/client/game/scripting/function_tables.cpp:1875]: No, was missing

**Review by mjkzy — APPROVED** [review]

File: src/client/game/scripting/function_tables.cpp [hunk 2]
```diff
@@ -1774,10 +1774,10 @@ namespace scripting
 		{"rotateto", 0x83DD}, // SP 0x1402B3200 MP 0x14037EF90
 		{"getlookaheaddir", 0x83DE}, // SP 0x000000000 MP 0x14044F710
 		{"getpathgoalpos", 0x83DF}, // SP 0x000000000 MP 0x14044FAD0
-		{"_meth_83e0", 0x83E0}, // SP 0x1402A15D0 MP 0x14036E1F0
+		{"setrocketcorpse", 0x83E0}, // SP 0x1402A15D0 MP 0x14036E1F0
 		{"setcorpsefalling", 0x83E1}, // SP 0x000000000 MP 0x140374840
 		{"setsurfacetype", 0x83E2}, // SP 0x14029FD00 MP 0x140358090
-		{"_meth_83e3", 0x83E3}, // SP 0x1402A0C60 MP 0x140369520
+		{"aIiphysicstrace", 0x83E3}, // SP 0x1402A0C60 MP 0x140369520
```
Comment (fedddddd) [inline src/client/game/scripting/function_tables.cpp:1780]: lowercase

**Review by fedddddd — CHANGES_REQUESTED** [review]

File: src/client/game/scripting/function_tables.cpp (hunk 2 above)
Comment (DoktorSAS) [reply src/client/game/scripting/function_tables.cpp]: Fixed

**Pull request merged**